omod/pom.xml

```diff
@@ -39,7 +39,7 @@
 		<dependency>
 			<groupId>org.openmrs.module</groupId>
 			<artifactId>hospitalcore-api</artifactId>
-			<version>1.5.3</version>
+			<version>1.0.0-SNAPSHOT</version>

```

Comment:
Put global references in the main pom file and just reference them in other subprojects

Reply:
Done